aiowebostv/webos_client.py

```diff
@@ -295,10 +263,10 @@ async def _closeout_tasks(
         self.connection = None
         self.input_connection = None
         self.do_state_update = False
-        self._clear_tv_states()
+        self.tv_state.clear()
```

Comment:
This is a change from previous implementation, the states is cleared upon connect and not upon disconnect so that in case a client was disconnected this data is still available to be used in diagnostics